modMcf/src/org/aion/mcf/config/CfgApiNrg.java

```diff
@@ -0,0 +1,137 @@
+/*******************************************************************************
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ *
+ ******************************************************************************/
+package org.aion.mcf.config;
+
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import javax.xml.stream.XMLStreamWriter;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+/**
+ * @author ali sharif
+ */
+public final class CfgApiNrg {
+    private long defaultPrice;
+    private long maxPrice;
+
+    CfgApiNrg() {
+        // recommend setting the defaultPrice to a safe-low known nrg price accepted by most miners on the network
+        // (this value fluctuates over time, depending on network conditions)
+        this.defaultPrice = 1_000_000L; // 1E6 AION
```

Comment:
These two values may warrant further discussion, this is also related to the price we should set as our minimum in the pool.

Reply:
TBD closer to mainnet launch. Adding the config parameters to the default generated config file.